test-suite/shared/src/test/scala/org/scalajs/testsuite/javalib/lang/StringTest.scala

```diff
@@ -279,9 +283,9 @@ class StringTest {
     assertEquals(s.length - 1, s.offsetByCodePoints(s.length - 1, 0))
     assertEquals(s.length, s.offsetByCodePoints(s.length, 0))
 
-    assertThrows(classOf[IndexOutOfBoundsException], s.offsetByCodePoints(-3, 4))
-    assertThrows(classOf[IndexOutOfBoundsException], s.offsetByCodePoints(6, 18))
-    assertThrows(classOf[IndexOutOfBoundsException], s.offsetByCodePoints(30, 2))
+    assertThrows(classOf[IndexOutOfBoundsException], s.offsetByCodePoints(-3, -4))
```

Comment:
Why the negative sign on `-4`? Wouldn't it make more sense to test only the `index` is negative case with this one?